components/service/pocket/src/test/java/mozilla/components/service/pocket/PocketStoryTest.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket
+
+import mozilla.components.service.pocket.PocketStory.PocketRecommendedStory
+import mozilla.components.service.pocket.PocketStory.PocketSponsoredStory
+import mozilla.components.service.pocket.helpers.assertConstructorsVisibility
+import mozilla.components.support.test.mock
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import kotlin.reflect.KVisibility
+
+class PocketStoryTest {
+    @Test
+    fun `GIVEN PocketSponsoredStory THEN it should be publicly available`() {
+        assertConstructorsVisibility(PocketSponsoredStory::class, KVisibility.PUBLIC)
+    }
+
+    @Test
+    fun `GIVEN PocketRecommendedStory THEN it should be publicly available`() {
+        assertConstructorsVisibility(PocketRecommendedStory::class, KVisibility.PUBLIC)
+    }
+
+    @Test
+    fun `GIVEN a PocketRecommendedStory WHEN it's title is accessed from parent THEN it returns the previously set value`() {
+        val pocketRecommendedStory = PocketRecommendedStory(
+            "testTitle", "", "", "", "", 0, 0
```

Comment:
Can you add the named parameter for these? Just easier for the future if we ever reorder or add another parameter.